civiform/civiform
# When unsetting universal on a question, show a confirmation modal

### Description

This PR adds a modal to confirm an admin would like to unset the Universal setting on a question. The modal shows when the admin is editing a question currently set to "Universal" and removing the "Universal" tag. This work is hidden behind the `UNIVERSAL_QUESTIONS` feature flag.

### Checklist

#### General



#### New Features


### Instructions for manual testing

1. Run the app, login as an admin, and make sure the `UNIVERSAL_QUESTIONS` feature flag is enabled in the settings panel
2. Navigate to question list and edit a question
3. Set the question as a "Universal" question by toggling the "Universal" toggle to on
4. Click "Update" (modal should not show)
5. Edit the same question
6. Toggle the "Universal" toggle to "off"
7. Click "Update" (modal should show)
8. Click the X or "Cancel" to close the modal and return to the edit page
9. Click "Update" again
10. Click "Remove from Universal Questions"
11. You should be redirected to the question list page and the Universal Question Badge should be gone

### Issue(s) this completes

Fixes #5859
